packages/cashier/src/pages/withdrawal/withdrawal-crypto-form/withdrawal-crypto-form.tsx

```diff
@@ -170,10 +175,12 @@ const WithdrawalCryptoForm = observer(() => {
                                 />
                             </div>
                             <CryptoFiatConverter
+                                arrow_icon_direction={arrow_icon_direction}
```

Comment:
can we use the updated currencies in `from_currency` and `to_currency` instead of passing this?

Reply:
hmmm i dont think we should, the reason being is because those values dont switch depending on which input the user is selecting. It will only just be used what currency each of the input boxes should display. But the arrow_icon_direction does keep track of which input the user has selected... hence why we decided to just keep track of the arrow direction since it already tracks which input its selecting.

Reply:
yuppp what we did was we brought the state to the parent where we can track it there, and we passed the state values to CryptoFiatConverter